sections/featured-blog.liquid

```diff
@@ -29,6 +29,14 @@
     assign posts_exceed_limit = true
     assign posts_displayed = section.settings.post_limit
   endif
+
+  if settings.spacing_grid_horizontal == 0 and settings.spacing_grid_vertical == 0 and settings.card_style == 'card'
+    assign collapsed_class = 'grid--collapsed grid--collapsed-x grid--collapsed-y'
+  elsif settings.spacing_grid_horizontal == 0
+    assign collapsed_class = 'grid--collapsed grid--collapsed-x'
+  elsif settings.spacing_grid_vertical == 0 and settings.card_style == 'card'
+    assign collapsed_class = 'grid--collapsed grid--collapsed-y'
+  endif
```

Comment:
So the problem is blog cards and collection cards in standard style can also render as text-only, which is essentially the same as the card style. A typical standard card with an image doesn't need to vertically collapse (which is why I was making that exception in the grid class conditionals). The mix of card types here makes it mostly impossible to know which type of card is going to be touching which type of card, especially considering the difference in heights.

After discussing with UX we've decided to always collapse the borders regardless of card type so we can make this case look slightly better. However this will not work for the corner collapsing logic. Bottomline is standard cards, especially without images, are not what we're targeting with this feature.